openstf/model/train.py

```diff
@@ -144,14 +204,37 @@ def train_model_for_specific_pj(
     train_data, validation_data, test_data = split_data_train_validation_test(
         clean_data_with_features
     )
+    return {
+        "train_data": train_data,
+        "validation_data": validation_data,
+        "test_data": test_data,
+    }
```

Comment:
I think I like the idea of using data structures for these kind of things. I'm wondering if there is not already something available out of the machine learning tools that we already use? If not, I'm not such a fan of dictionairies, I would prefer a more rigid data container like a `dataclass` or `namedtuple` the main advantage of these containers is that they are documented (you just look up their definition and know which attributes do exists) and you can make then non extendible.